tests/jobs/misfit_preprocessor/test_integration.py

```diff
@@ -48,7 +44,7 @@ def test_misfit_preprocessor_main_entry_point_gen_data(monkeypatch):
     # call_args is a call object, which itself is a tuple of args and kwargs.
     # In this case, we want args, and the first element of the arguments, which
     # again is a tuple containing the configuration which is a list of configs.
-    assert len(run_mock.call_args[0][0]) == 47, "wrong number of clusters"
+    assert len(list(run_mock.call_args)[0][0]) == 47, "wrong number of clusters"
```

Comment:
Don't understand why we all of a sudden needs `list`, but okay..? :P

Reply:
A pylint issue, could opt to ignore the issue instead if you would prefer? `unsubscriptable` error

Reply:
I would almost argue that it's better to include the `ignore unsubscriptable` - as that will highlight all workarounds necessary with pylint, instead of hiding it in the code.

But I'll leave it up to you to decide